jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/es/ESManager.java

```diff
@@ -404,6 +404,7 @@ public static void configureIndexAndTypes(String ds, List<Map<String, String>> e
                 StringBuilder payLoad = new StringBuilder(_payLoad);
                 payLoad.append("\"dynamic\": true,");
                 payLoad.append("\"properties\": {");
+                payLoad.append("\"createdDate\": {\"type\": \"date\",\"format\": \"yyyy-MM-dd HH:mm:00Z||yyyy-MM-dd HH:mm:ss.SSSSSS||strict_date_optional_time||epoch_millis\"},");
```

Comment:
@plyubich this is generic and and is part of creating all index if they do not already exist. It will not run for the already existing asset type for an asset group.Also,  this attribute is already mapped as date for other asset type as part of violation, this change is needed for assets that have createdDate as an attribute so that es can support the date format from collector and correctly parse as date. Please connect for further clarification.